components/api_request_helper/api_request_helper.cc

```diff
@@ -192,6 +212,7 @@ std::unique_ptr<network::SimpleURLLoader> APIRequestHelper::CreateLoader(
   if (!payload.empty()) {
     url_loader->AttachStringForUpload(payload, payload_content_type);
   }
+  url_loader->SetTimeoutDuration(base::Seconds(30));
```

Comment:
Do we actually need to change current behavior ?

Reply:
Debug thing :facepalm: